### PR TITLE
[p5.js hi]: Examples Translated

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**

### CODE REVIEW

File: src/content/examples/hi/11_3D/00_Geometries/description.mdx [hunk 1]
```diff
@@ -1,15 +1,12 @@
----
+---
 featuredImage: "../../../images/featured/11_3D-00_Geometries-thumbnail.png"
-title: Geometries
-one-line-description: Draw 3D shapes, including a custom model.
+title: ज्योमेट्री
```
Comment (TanviKumar) [inline src/content/examples/hi/11_3D/00_Geometries/description.mdx:3]: Use "geometry" the same spelling across all pages. Do take a look

File: src/content/examples/hi/15_Math_And_Physics/01_Soft_Body/description.mdx [hunk 2]
```diff
@@ -1,12 +1,10 @@
 ---
 featuredImage: "../../../images/featured/15_Math_And_Physics-01_Soft_Body-thumbnail.png"
-title: Soft Body
-one-line-description: Simulate the physics of a soft body accelerating toward the mouse.
+title: कोमल शरीर
```
Comment (TanviKumar) [inline src/content/examples/hi/15_Math_And_Physics/01_Soft_Body/description.mdx:3]: Use same translation for soft across file

File: src/content/examples/hi/14_Loading_And_Saving_Data/02_Table/description.mdx [hunk 3]
```diff
@@ -1,15 +1,12 @@
 ---
 featuredImage: "../../../images/featured/14_Loading_And_Saving_Data-02_Table-thumbnail.png"
-title: Table
-one-line-description: Store data as comma-separated values.
+title: मेज़
```
Comment (TanviKumar) [inline src/content/examples/hi/14_Loading_And_Saving_Data/02_Table/description.mdx:3]: Use suuchi instead, this means physical table

File: src/content/examples/hi/15_Math_And_Physics/03_Smoke_Particle_System/description.mdx [hunk 4]
```diff
@@ -1,22 +1,11 @@
...
-class, including the
-<a href="https://p5js.org/reference/#/p5/createVector">createVector()</a>
-function. The various calculations for updating particles'
-positions and velocities are performed with p5.Vector methods.
-
-The particle system is implemented as a
-<a href="https://p5js.org/reference/#/p5/class">class</a>, which contains an array of
-objects (of class Particle).
\ No newline at end of file
+स्मोक पार्टिकल सिस्टम डेमो, प्रसंस्करण के लिए डैन शिफमैन के मूल <a href="https://natureofcode.com/book/chapter-4-particle-systems/">उदाहरण</a> पर आधारित है। कोड का उपयोग करता है
```
Comment (TanviKumar) [inline src/content/examples/hi/15_Math_And_Physics/03_Smoke_Particle_System/description.mdx:8]: Some a tags have been translated and others haven't, do take a look across all files

**Review by TanviKumar — CHANGES_REQUESTED** [review]

File: src/content/examples/hi/14_Loading_And_Saving_Data/02_Table/description.mdx (hunk 3 above)
Comment (SanjaySinghRajpoot) [reply src/content/examples/hi/14_Loading_And_Saving_Data/02_Table/description.mdx:3]: Found a better alternative तालिका

File: src/content/examples/hi/15_Math_And_Physics/03_Smoke_Particle_System/description.mdx (hunk 4 above)
Comment (SanjaySinghRajpoot) [reply src/content/examples/hi/15_Math_And_Physics/03_Smoke_Particle_System/description.mdx:8]: Yes, some of the tags have been translated as they were simple to understand and their meaning was the same in hindi. A few of the tags are not translated as the meaning was different in Hindi.

File: src/content/examples/hi/15_Math_And_Physics/01_Soft_Body/description.mdx (hunk 2 above)
Comment (SanjaySinghRajpoot) [reply src/content/examples/hi/15_Math_And_Physics/01_Soft_Body/description.mdx:3]: ok, I will check this

File: src/content/examples/hi/11_3D/00_Geometries/description.mdx (hunk 1 above)
Comment (TanviKumar) [reply src/content/examples/hi/11_3D/00_Geometries/description.mdx:3]: @SanjaySinghRajpoot could you take a look at this one

**Review by TanviKumar — APPROVED** [review]

**Review by Divyansh013 — APPROVED** [review]

**Comment by TanviKumar** [comment]
@Qianqianye this PR can be merged

**Comment by Qianqianye** [comment]
Thanks, could you resolve the conflicts? @SanjaySinghRajpoot @Divyansh013 @TanviKumar

**Comment by SanjaySinghRajpoot** [comment]
@Qianqianye I have fixed the merge conflicts, you can merge the PR now.

**Comment by TanviKumar** [comment]
@Qianqianye @limzykenneth the PR can be merged

**Comment by limzykenneth** [comment]
@Qianqianye @outofambit @stalgiag Considering the content freeze, is the plan to have interim content updates merge onto a different branch than `main`? Or we can hold off on merging this for now too. I'll fix the test too shortly.

**Comment by stalgiag** [comment]
> @Qianqianye @outofambit @stalgiag Considering the content freeze, is the plan to have interim content updates merge onto a different branch than `main`? Or we can hold off on merging this for now too. I'll fix the test too shortly.

I made a [localization-updates](https://github.com/bocoup/p5.js-website/tree/localization-updates) branch. You can use that for localization changes during this week and we can merge into `main` next week after this QA cycle closes.

**Comment by limzykenneth** [comment]
Merged into [localization-updates](https://github.com/bocoup/p5.js-website/tree/localization-updates) branch.

**Pull request closed without merging**